.storybook/preview.js

```diff
@@ -16,4 +25,14 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  darkMode: {
+    dark: {
+      ...themes.dark,
+      appContentBg: tokens.dark.surface['surface-primary'].value,
+    },
+    light: {
+      ...themes.normal,
+      appContentBg: tokens.light.surface['surface-primary'].value,
+    },
```

Comment:
The theme plugin wasn't playing nicely with css vars so I'm using the js version of these tokens.